common/test/acceptance/pages/lms/edxnotes.py

```diff
@@ -34,9 +34,9 @@ def _get_element_text(self, selector):
             return None
 
 
-class EdxNotesPageGroup(NoteChild):
+class EdxNotesChapterGroup(NoteChild):
     """
-    Helper class that works with note groups on Note page of the course.
+    Helper class that works with chapter (section) grouping of notes in the Course Structure view on the Note page.
```

Comment:
Really? I have never heard it and don't see the reason why. If pylint/pep8 doesn't care, I certainly don't.